docs/source/model_design/risk_factors.rst

```diff
@@ -531,9 +531,206 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
+The population attributable fraction is a quantification of the proportion of 
+of a given cause outcome, such as cases, deaths, or DALYs, that could be 
+eliminated by removing a risk exposure.
+
+..todo::
+	Fill in PAF overview
+
+Bias in PAF Calculation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The PAF can be calculated using the following formula:
+
+.. math::
+	:label: eq1
+
+	\text{PAF}_{pop}=\frac{p_c(RR_{adj}-1)}{RR_{adj}}
+
+In which we define :math:`p_c` to be the proportion of cases (individuals who 
+possess the outcome of interest) that are exposed, and :math:`RR_{adj}` has been adjusted for confounding and effect modification.
+
+There is the a second PAF equation, which can be used *in the absence of 
+confounding or effect modification:*
+
+.. math:: \text{PAF}_{pop}=\frac{p_p(RR_{cr}-1)}{1+p_p(RR_{cr}-1)} =\frac{p_p(RR_{adj}-1)}{1+p_p(RR_{adj}-1)}
+	:label: eq2
+
+Note that here, the crude relative risk :math:`(RR_{cr})` is equivalent to the adjusted :math:`(RR_{adj})`. We define :math:`p_p` to be the proportion of the entire 
+population that is exposed.
+
+This is typically easier to conceptualize if we break the population down as
+follows:
+
+.. list-table:: Exposure x Cases
+	:widths: 1 1 1
+	:header-rows: 1
+	:stub-columns: 1
+	:align: center
+
+	* - 
+	  - Cases
+	  - Non-cases
+	* - Exposed
+	  - a
+	  - b
+	* - Unexposed
+	  - c
+	  - d
+
+Observe that the above table is a full partition of our population. We can see 
+then that the proportion of cases that are exposed is given by:
+
+..	math:: p_c=\frac{a}{a+c}
+
+And the proportion of the entire population that is exposed is given by:
+
+.. math:: p_p = \frac{a+b}{a+b+c+d}
+
+It can be shown that when the fraction of cases in the unexposed times the 
+relative risk :math:`\left( \frac{c}{c+d} \cdot RR_{adj} \right)` equals the fraction 
+of cases in the exposed :math:`\left( \frac{a}{a+b} \right)`, i.e., when there 
+are no confounders, equation (1) equals equation (2).
```

Comment:
What about effect modification? Do we have to assume there is no effect modification as well as no confounding in order for (1) to equal (2)?